src/universal/redux/rootDuck.js

```diff
@@ -0,0 +1,21 @@
+
+export const RESET = '@@root/RESET';
+
+export default (appReducer) => (state, action) => {
+  if (action && action.type === RESET) {
+    const {whitelist} = action.payload;
+    const newState = {};
+    whitelist.forEach((keeper) => {
+      if (keeper in state) {
+        newState[keeper] = state[keeper];
```

Comment:
Just making sure shallow is enough here: https://twitter.com/dan_abramov/status/688087202312491008